app/controllers/api/v1/books_controller.rb

```diff
@@ -39,6 +39,15 @@ def destroy
         end
       end
 
+      def book_data_by_isbn
```

Comment:
Al ser books_controller se podría entender que es basado en books y la ruta quedaría mas explicita, ejem:
`api/v1/books/by_isbn`

```suggestion
      def by_isbn
```